packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenCard.tsx

```diff
@@ -104,7 +104,8 @@ export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
       {/* Delete button */}
       <div className="PersonalAccessTokenCard__DeleteButton">
         <Button
-          variant="default"
+          variant="outlined"
+          color="primary"
```

Comment:
before | after
:---:|:---:
<img width="1394" alt="before_AccessTokenPage" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/c5ac01da-fc1d-499d-bfc8-0289731be8b9"> | <img width="1398" alt="feature_AccessTokenPage" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/37fc2bef-7799-45e3-9854-2e670f6d3ff8">

Reply:
Good call, done in the next commit